### PR TITLE
[RPS-133] Prevent saving publication with blank related link field

### PR DESCRIPTION
**Pull request by ngta1**

### CODE REVIEW

File: acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/ContentPage.java [hunk 1]
```diff
@@ -103,11 +103,14 @@ public void populatePublicationSummary(final String publicationSummary) {
     public AttachmentsSection getAttachmentsSection() {
         return new AttachmentsSection(helper, getWebDriver());
     }
-
```
Comment (wojtek-oledzki) [inline acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/ContentPage.java]: please re-add blank lines between functions :)

File: repository-data/application/src/main/resources/hcm-config/configuration/frontend/cms-validators.yaml [hunk 2]
```diff
@@ -18,3 +18,7 @@ definitions:
       jcr:primaryType: frontend:plugin
       plugin.class: uk.nhs.digital.ps.BlankStaticDropdownSelectionFieldValidator
       fieldDisplayName: Granularity
+
+    /hippo:configuration/hippo:frontend/cms/cms-validators/blank-related-link-field-forbidden:
```
Comment (wojtek-oledzki) [inline repository-data/application/src/main/resources/hcm-config/configuration/frontend/cms-validators.yaml]: can you rename this one to "publicationsystem-blank-..."

And could you also rename "granularity" and "attachment" too?

File: acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/ContentPage.java (hunk 1 above)
Comment (ngta1) [reply acceptance-tests/src/test/java/uk/nhs/digital/ps/test/acceptance/pages/ContentPage.java]: Took out one line too many during rebase :-)  Will re-add.

File: repository-data/application/src/main/resources/hcm-config/configuration/frontend/cms-validators.yaml (hunk 2 above)
Comment (ngta1) [reply repository-data/application/src/main/resources/hcm-config/configuration/frontend/cms-validators.yaml]: Shall I rename to:
"publicationsystem-blank-granularity"
"publicationsystem-blank-attachment"
"publicationsystem-blank-related-link"

or without that "...-blank-..." which will keep it consistent with the other two validators:
"publicationsystem-summary"
"publicationsystem-title"

**Comment by wojtek-oledzki** [comment]
I Like

I Like

**Pull request merged**